ProjectLighthouse.Localization/BaseLayout.resx

```diff
@@ -49,6 +49,10 @@
         <comment>A button on the header that takes you to a list of authentication attempts.</comment>
     </data>
 
+    <data name="header_login" xml:space="preserve">
+        <value>Login</value>
+        <comment>A button on the header that lets you log in.</comment>
+    </data>
     <data name="header_loginRegister" xml:space="preserve">
```

Comment:
Since you're removing the header_home and header_profile TranslatableStrings the respective resource sections should be deleted too